tests/test_files/tournamentWithRounds.yaml

```diff
@@ -0,0 +1,27 @@
+players:
+- alice
+- bob
+- charlie
+- dave
+rounds:
+  Round 1:
+  - w: alice
```

Comment:
I would expand w, b, and r to `white_player`, `black_player` and `result` for clarity.

Reply:
no problem, I will change it